src/lib/offlineMutationQueue.ts

```
@@ -206,23 +206,24 @@ export async function replayOfflineMutations(): Promise<{
+if (item.action === "add") {
+  const { data, error } = await supabase.rpc("join_event_or_waitlist", {
+    p_event_id: item.payload.eventId,
+    p_user_id: userId,
+    p_is_anonymous: false,
+    p_resume_path: null,
+    p_referred_by: null,
+  });
+
+  isSuccess = !error && data?.success === true;
+} else {
+  const { data, error } = await supabase.rpc("cancel_event_rsvp", {
```

ESLint

[error] Insert `········`

(prettier/prettier)